### PR TITLE
Add members tab to cw20 and cw721 DAOs and add interactive controlling distribution interface

### LINKED ISSUE #1107: Missing Visibility of All NFTs in NFT-based DAOs
**Issue:** 
Missing Visibility of NFTs in NFT-based DAOs

**Steps to Reproduce:**
Visit an NFT-based DAO
Attempt to view the NFTs within the DAO or collection

**Expected Result:**
The NFT-based DAO should have a "Members" or "Collection" tab to show all the NFTs within the collection.

**Actual Result:**
The only option currently available is a "Staked NFTs" tab, which does not provide visibility into the NFTs within the DAO or collection.
image

### PR DESCRIPTION
**Pull request by NoahSaso**
Resolves #1107 

This adds a members tab to CW20-based DAOs, adds a members tab to CW721-based DAOs, and changes the `Staked NFTs` tab for CW721-based DAOs into an `NFT Collection` tab that shows all NFTs and allows filtering by all/staked/unstaked. It also standardizes members tabs to show top voting power holders with a chart.

## Members tab for cw20 and cw721 DAOs

Screenshot 2023-02-23 at 4 50 00 PM
Screenshot 2023-02-23 at 4 47 28 PM
Screenshot 2023-02-23 at 4 47 37 PM
Screenshot 2023-02-23 at 4 47 47 PM

#### Other hides when the selection encompasses 100% of voting power

Screenshot 2023-02-23 at 4 42 06 PM

## NFT Collection Tab
Screenshot 2023-02-23 at 4 48 27 PM
Screenshot 2023-02-23 at 4 48 39 PM
Screenshot 2023-02-23 at 4 48 45 PM

### CODE REVIEW

**Review by JakeHartnell — APPROVED** [review]

File: packages/i18n/locales/en/translation.json [hunk 1]
```diff
@@ -578,6 +578,9 @@
     "mintNftDescription": "Create a new NFT.",
     "name": "Name",
     "newMemberExplanation": "To add a new member, the DAO must pass a proposal.",
+    "nftCollectionExplanation_all": "These are all of the NFTs in the collection that controls the DAO. Staking an NFT increases the staker's voting power.",
```
Comment (elsehow) [inline packages/i18n/locales/en/translation.json:581]: > In this DAO, voting share is determined by staking NFTs from a collection. This is the collection members stake to gain voting power in this DAO.

File: packages/i18n/locales/en/translation.json [hunk 2]
```diff
@@ -578,6 +578,9 @@
...
+    "nftCollectionExplanation_staked": "These NFTs have been staked by members of the DAO to earn voting power.",
+    "nftCollectionExplanation_unstaked": "These NFTs have not been staked and provide no voting power.",
```
Comment (elsehow) [inline packages/i18n/locales/en/translation.json:583]: > These NFTs would provide voting power if they were staked, but no one is currently staking them.

File: packages/i18n/locales/en/translation.json [hunk 3]
```diff
@@ -619,6 +622,7 @@
     "pending": "Pending",
     "pendingRewards": "Pending rewards",
     "percentVotingPower": "{{percent}} voting power",
+    "percentagesShownAreVotingPower": "Percentages shown are voting power.",
```
Comment (elsehow) [inline packages/i18n/locales/en/translation.json:625]: > Percentages represent voting power.

**Review by elsehow — APPROVED** [review]
I love this as you know. A few minor notes.

**Pull request merged**